firebase.json

```diff
@@ -1,6 +1,14 @@
 {
-  "hosting": {
-    "public": "out",
-    "ignore": ["firebase.json", "**/.*", "**/node_modules/**"]
-  }
+  "hosting": [
+    {
+      "target": "root",
+      "public": "out",
+      "ignore": ["firebase.json", "**/.*", "**/node_modules/**"]
```

Comment:
The target `root` is intended for the template branch of the monorepo, currently nothing like that exists yet tho. We can associate a site to it later on.